pkg/api/types.go

```diff
@@ -769,11 +768,6 @@ func (a *AgentPoolProfile) HasDisks() bool {
 	return len(a.DiskSizesGB) > 0
 }
 
-// IsAcceleratedNetworkingEnabled returns true if the customer enabled Accelerated Networking
-func (a *AgentPoolProfile) IsAcceleratedNetworkingEnabled() bool {
```

Comment:
I was going to write unit tests for this but then I realized there's no point in having this function so I removed it